### ISSUE TITLE
What does "Block this site" do?

### ISSUE THREAD

**Issue by orschiro** [GaurangTandon/ProKeys#196]
Dear @GaurangTandon,

What a great extension. Thank you so much! This really speeds up reusable snippets. :-) 

Question. Couldn't figure out what the context menu entry "Block this site" exactly does. 

Could you clarify?

Thanks!

**Comment by GaurangTandon** [comment]
Thanks for your feedback Robert! :-)

The "Block this site" facility is provided for you to list those websites where you don't wish ProKeys to run For example, you may be on a company website where they provide their own custom textbox which interferes with ProKeys and makes it impossible to work with ProKeys enabled.  So, you can disable ProKeys on those specific websites.

You will also find the entire list of blocked sites in your Settings page. 

I hope this helps!

**Comment by GaurangTandon** [comment]
I am closing the issue. Please feel free to mention if you're more feedback. Thanks!

**Comment by GaurangTandon** [comment]
Yep, silly me, I guess that's better. Will incorporate this into the next release! :D

**Comment by orschiro** [comment]
Great, thanks so much! :-)

**Comment by GaurangTandon** [comment]
@JAIDEV123 please keep this in mind for the new options page.
